### ISSUE TITLE
SignedRequest class does not inherit ReportCard attributes

### ISSUE THREAD

**Issue by tomasholderness** [urbanriskmap/cognicity-cards#11]
- photo upload currently broken because ReportCard attributes are not passed to the getSignedURL function, specifically;

- ReportCard.id, and
the 'type' variable in the getSignedURL method.

I've added error handling (in 2b8ea8ac3c3b2b1d458b8c17067ae7978cc91940) here: https://github.com/urbanriskmap/cognicity-cards/commit/2b8ea8ac3c3b2b1d458b8c17067ae7978cc91940#diff-df3e58fc2015960b94028752a4cc6e4dR30 so that we can see this play out when you try and upload an image, but going to need some help with JavaScript inheritance to work out why we can't access the id variable (no getter defined)?

**Comment by tomasholderness** [comment]
Ping @matthewberryman -> FYI image upload broken until this is fixed.

**Comment by tomasholderness** [comment]
Update: 
if you log ReportCard.id at: https://github.com/urbanriskmap/cognicity-cards/blob/2b8ea8ac3c3b2b1d458b8c17067ae7978cc91940/src/services/signed-request.js#L11 
...you can see that it isn't set.

**Comment by ojha-url** [comment]
Can you try passing the id in the getSignedUrl function inside photo card?
Along with the image type.

**Comment by tomasholderness** [comment]
@ojha-url - ok trying that now.

**Comment by tomasholderness** [comment]
Few! This was compounded by:
1) API gateway not passing the new content-type header
2) No error handling in the cards if the signed URL returned a 400 response.

Now fixed!

@matthewberryman - I updated API gateway to add the header "content-type" to the cards/{cardid}/images endpoint. Should I be updating a doc/spec/swagger somewhere to reflect these changes, or can we generated from current API config as listed in API gateway?

Thanks.
